crates/eww/src/config/inbuilt.rs

```diff
@@ -81,17 +81,17 @@ macro_rules! define_magic_constants {
     }
 }
 define_magic_constants! { eww_paths,
-    // @desc EWW_CONFIG_DIR - Path to the eww configuration of the current process
+    // @desc EWW_CONFIG_DIR (Magic constant) - Path to the eww configuration of the current process
     "EWW_CONFIG_DIR" => DynVal::from_string(eww_paths.get_config_dir().to_string_lossy().into_owned()),
 
-    // @desc EWW_CMD - eww command running in the current configuration, useful in event handlers. I.e.: `:onclick "${EWW_CMD} update foo=bar"`
+    // @desc EWW_CMD (Magic constant) - eww command running in the current configuration, useful in event handlers. I.e.: `:onclick "${EWW_CMD} update foo=bar"`
     "EWW_CMD" => DynVal::from_string(
         format!("\"{}\" --config \"{}\"",
```

Comment:
```suggestion
        format!("{} --config {}",
```

Reply:
Yeah, that's what I thought, BUT. Try cloning eww into a folder named `eww with   spaces` for example. It doesn't work for me. The path is just cut from the first space to the end. These are kind of things that make want not to deal with bash ever again(

Reply:
yeah true, i forgot about that, nvm

Reply:
Why not use single quotes

Reply:
Same problems as with double quotes